src/strings/practice/1.js

```diff
@@ -6,13 +6,34 @@
 
 module.exports = function capFirstLowLast(str) {
 	'use strict';
-	// your code
+	let firstChar = str[0];
+	let firstCharUpper = firstChar.toUpperCase();
```

Comment:
you may change those two lines with one: `let firstCharUpper = str[0].toUpperCase();`

Reply:
done